app/controllers/employees_controller.rb

```diff
@@ -25,7 +24,7 @@ def update
   end
 
   def check_employee
-    return if current_employee.first_name == 'default' && current_employee.last_name == 'default'
+    return if current_employee.first_name.nil? && current_employee.last_name.nil?
```

Comment:
I think it could be moved to the policy, but this is my reflection ;)

Reply:
good catch @rniedziolka ! <3

Reply:
I took it to the policy :)